docs/guides/process-collector.md

```diff
@@ -0,0 +1,101 @@
+---
+title: 'The "process-collector" gadget'
+weight: 10
+---
+
+The process-collector gadget gathers information about running processes.
+
+Let's start this demo by creating a namespace:
+
+```
+$ kubectl create ns demo
+namespace/demo created
+```
+
+There is not any running process in the `demo` namespace now:
+
+```
+$ kubectl gadget process-collector -n demo
+NAMESPACE    POD    CONTAINER    COMM    PID
+```
+
+Create a pod on the `demo` namespace using the `nginx` image:
+
+```
+$ kubectl -n demo run mypod --image=nginx
+pod/mypod created
+```
+
+After the pod is running, we can try to get the list of running processes again:
+
+```
+$ kubectl gadget process-collector -n demo
+NAMESPACE    POD      CONTAINER    COMM     PID
+demo         mypod    mypod        nginx    34270
+demo         mypod    mypod        nginx    34308
+demo         mypod    mypod        nginx    34309
+demo         mypod    mypod        nginx    34310
+demo         mypod    mypod        nginx    34311
+demo         mypod    mypod        nginx    34312
+demo         mypod    mypod        nginx    34313
+demo         mypod    mypod        nginx    34314
+demo         mypod    mypod        nginx    34315
```

Comment:
I am not a fan of this example because the user could get distracted by all these identical processes (BTW, I did 😅) but it is indeed an example of use so it is ok for me.